Microtasks/GSoD-2020_Micro.md

```diff
@@ -0,0 +1,40 @@
+# Google Season of Docs - microtasks
+
+## Project overview
+
+The CHAOSS Diversity and Inclusion Badging Program began as a way to ensure that D&I metrics can have a significant impact on the ways that communities function. The aim of the CHAOSS D&I Badging program is to bring D&I metrics into use for propelling good D&I practices in different open source systems.
+
+This project will focus on framing documentation for the D&I Badging program and aligning it in such a way that it makes sense after translation. This work would focus on more community-facing docs related to Badging and also on existing Markdown files.
+
+## Submitting microtasks
+
+**All microtasks must be submitted as a response to this issue.**
+
+## Microtask 0:
+
+`Q 0` Review [JOSS documentation](https://joss.readthedocs.io/en/latest/) on submission and review and see how it is related to the current workflow.
+
+The Journal of Open Source Software is one of the primary references used for building out the CHAOSS Badging process. The goal is to compare how the current process works in comparison to the review process of JOSS.
+
+Reference:
+- [Joss Documentation](https://joss.readthedocs.io/en/latest/editing.html)
+
+`Q 1` Review the CII self-report system on projects and events and report schematic differences.
+
+Projects and events are very different, but how different are they in our program? List one thing you would change about either, or both.
+
+
+## Microtask 1:
+
+`Q 0` Submit a non-existent project to understand the workflow.
+
+Create a draft PR after adding a mock event in either the [Project Badging](https://github.com/badging/project-diversity-and-inclusion) or [Event Badging](https://github.com/badging/event-diversity-and-inclusion) repository. Record your observations as a part of the microtask.
+
+Reference:
+- [Project Badging](https://github.com/badging/project-diversity-and-inclusion)
+- [Event Badging](https://github.com/badging/event-diversity-and-inclusion) 
+
+`Q 1` Review a mock event created by someone else.
+
+If available, take the time to review an event that someone else has posted. If no fake events have been posted yet, put `N/A` here and post it when a mock submission is available.
+
```

Comment:
Here are a couple of mock events that were created during GSoC application period:
  - https://github.com/badging/event-diversity-and-inclusion/pull/1
  - https://github.com/badging/diversity-and-inclusion/pull/2